km/km.h

```diff
@@ -140,6 +140,14 @@ typedef enum {
    PAUSED       // Paused in km_vcpu_handle_pause
 } __attribute__((__packed__)) km_vcpu_state_t;
 
+typedef enum {
+   SNAP_STATE_RUNNING,
+   SNAP_STATE_PAUSED,
+   SNAP_STATE_RUNHOOK_CREATE,
+   SNAP_STATE_RUNHOOK_RESTORE,
+   SNAP_STATE_RUNHOOK_RESTORE_LIVE,
+} km_snap_state_t;
```

Comment:
Can you explain why these states are not part of km_vcpu_state_t with a comment here?

Reply:
ok